eldritch/expansions/seaside/abilities.py

```diff
@@ -0,0 +1,114 @@
+import typing
+
+from eldritch import assets
+from eldritch import characters
+from eldritch import events
+from eldritch import values
+
+if typing.TYPE_CHECKING:
+  from eldritch.eldritch import GameState
+
+
+class AbnormalFocus(assets.Asset):
+  def __init__(self):
+    super().__init__("Abnormal Focus")
+
+  def get_interrupt(self, event, owner: characters.BaseCharacter, state):
+    if isinstance(event, events.Upkeep) and event.character == owner:
```

Comment:
I actually have no idea when this is supposed to happen. It could be at the start of the upkeep phase. But then the complete FAQ says that events that happen during upkeep happen during the "upkeep actions" part of upkeep, which comes after the "refresh stuff" part of upkeep. This could also mean that the Spy would have a very hard time casting spells that can only be cast during upkeep? This is very confusing.

Reply:
Plenty of people have argued about this, without settling on a firm conclusion. 
- https://boardgamegeek.com/thread/530872/upkeep-adjusting-skill-sliders-and-casting-spells/page/2
- https://boardgames.stackexchange.com/questions/8842/when-during-your-upkeep-can-you-cast-upkeep-spells

One resolution could be that the spy is special, and may move her sliders at the beginning of upkeep. 
> Instead, at the start of each [Upkeep Phase](http://www.arkhamhorrorwiki.com/Upkeep_Phase), move her sliders all the way to the left. She may then shift her sliders to the right a total of five shifts.